src/index.spec.js

```diff
@@ -498,6 +593,144 @@ describe('Dropzone', () => {
       expect(dropRejectedSpy.callCount).toEqual(0);
     });
 
+    it('accepts files on the innermost Dropzone when we drop into it', () => {
```

Comment:
The innermost dropzone is the child Dropzone component. The outermost dropzone is the parent Dropzone component.